### PR TITLE
[fullstack] improvements to scanner

### PR DESCRIPTION
**Pull request by WilhelmWillie**
* Fixes a 500 error that happened when a non-admin accessed scan page
* Adds 'identify" action to scanner 
* Fixes stupid last scanned code bug

### CODE REVIEW

File: pages/checkin.tsx [hunk 1]
```diff
@@ -41,8 +45,19 @@ const CheckinResult = ({ result, resetResults }) => {
       });
 
       if (!assignRequest.error) {
-        alert("Successfully assigned hacker QR");
-        resetResults();
+        const checkinRequest = await liveDispatchFetch({
+          actionId: "checkin",
+          qrCodeId: qrInputValue
+        });
```
Comment (abigger87) [inline pages/checkin.tsx]: We have to use async/await due to fetcher's return type being async func?

Comment (WilhelmWillie) [reply pages/checkin.tsx]: yup

File: api/hackerLive.js [hunk 2]
```diff
@@ -157,30 +157,30 @@ router.get("/tasks", async (req, res) => {
   return res.json({ success: tasks });
 });
 
-router.get("/houseInfo/:id", async (req, res) => {
-  const houseId = req.params.id;
```
Comment (WilhelmWillie) [inline api/hackerLive.js]: switched the order of `/houseInfo/:id` and `/houseInfo/list` because `/:id` being before `/list` messed things up

Comment (saresend) [reply api/hackerLive.js]: Oh does it do in order eval?

Comment (WilhelmWillie) [reply api/hackerLive.js]: yeah i think so

**Pull request merged**